rest/route/task.go

```diff
@@ -298,6 +298,59 @@ func (tep *taskExecutionPatchHandler) Run(ctx context.Context) gimlet.Responder
 	return gimlet.NewJSONResponse(taskModel)
 }
 
+// GET /tasks/{task_id}/display_task
+
+type displayTaskName struct {
```

Comment:
Can we export this so we can reuse it in the client code? Idk if there will be a circular dependency, but if so, we can put it in the rest/model package.

Reply:
I'm reluctant to do this because Evergreen packages tend to have confusing, tightly-coupled dependency trees that just barely avoid cycles, so I don't want to introduce a dependency between the REST client and REST route package.

One possible alternative would be to just return the payload as a plaintext string since it's just a name.

Reply:
I am happy with keeping it as an alternative, or moving it to the rest/model package since that is already imported by rest/client.

Reply:
I just changed it to return a regular string since the payload is simple enough.